server/apps/publish/archive_publish.py

```diff
@@ -258,6 +259,8 @@ def _publish_package_items(self, package, updates):
                     package_item = super().find_one(req=None, _id=guid)
 
                 subscribers = self._get_subscribers_for_package_item(package_item)
+                PackageService().update_field_in_package(updates, package_item[config.ID_FIELD],
+                                                         'version', package_item['_current_version'])
```

Comment:
Shouldn't be `_current_version`? Please use the constant `config.VERSION` reason being if we change the name then this will fail.